src/esp_littlefs.c

```diff
@@ -20,6 +19,16 @@
 #include <sys/lock.h>
 #include <sys/param.h>
 #include <unistd.h>
+#include "esp_idf_version.h"
+
+#if ESP_IDF_VERSION >= ESP_IDF_VERSION_VAL(5, 0, 0)
+#include "spi_flash_mmap.h"
+#define pcTaskGetTaskName pcTaskGetName
```

Comment:
pcTaskGetName is used since v4.3, so one option here would be to rephrase this as follows:
```c++
#if ESP_IDF_VERSION < ESP_IDF_VERSION_VAL(4.3.0)
#define pcTaskGetName pcTaskGetTaskName
#endif
```
and replace the usage of pcTaskGetTaskName with pcTaskGetName.

This way when IDF v4.2 becomes EOL, it will be easier to notice and remove this fragment.

Reply:
addressed in [5797d8b](https://github.com/joltwallet/esp_littlefs/pull/91/commits/5797d8ba7b69b63b678ac6866fc5366f9c2bf163)